### DISCUSSION TITLE
mmdetection能不能去掉neck，我想做个对比实验

### DISCUSSION THREAD

**Discussion by talent-ywh** [open-mmlab/mmdetection#9689] [Q&A] [answered]
mmdetection能不能去掉neck，我想做个对比实验

**Comment by Li-Qingyun** [comment]
可以的

**Reply by talent-ywh** [reply]
如何去掉啊，去掉报错了

**Reply by Li-Qingyun** [reply]
直接删除对应的neck的config字段就算去掉了，head可能需要根据变化相应调整一些参数，解决报错是肯定要解决的

**Comment by vansin** [comment]
ssd 好像没有neck

**Reply by Li-Qingyun** [reply]
对对，好多都没有neck，早期detr也是没有neck的。而且代码里都是会判断   if self.with_neck:   x = self.neck(x)

**Comment by ZwwWayne** [comment, marked as answer]
Simply setting `neck=None`, then set the channels to be consistent between backbone and the head.

**Reply by talent-ywh** [reply]
Ok, thank you. I'll try
